homeassistant/components/ffmpeg_noise/binary_sensor.py

```diff
@@ -26,6 +26,7 @@
         vol.Required(CONF_INPUT): cv.string,
         vol.Optional(CONF_INITIAL_STATE, default=DEFAULT_INIT_STATE): cv.boolean,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_UNIQUE_ID): cv.string,
```

Comment:
Is this not used anywhere in the binary sensor definition?

Reply:
I don't see the unique ID defined for binary sensors. I noticed that it is added for specify integrations (like template, lock). This is the main reason for this PR.